### PR TITLE
RPM spec update for openSUSE:Factory linter compliance 3821

### PR DESCRIPTION
**Pull request by sandrew**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
@lagartoflojo will review your pull request :dancers: check https://trello.com/c/GuKINtsG/3821-3-connect-submit-to-factory
  @lagartoflojo: Please review this pull request using our guidelines:
  * test for acceptance criteria / functionality
  * check if the new code is covered with tests
  * check for unintended consequences
  * encourage usage of the boyscout rule
  * make sure the code is architected in the best way
  * check that no unnecessary technical debt got introduced
  * make sure that no unnecessary FIXMEs or TODOs got added

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-Connect-Prophet/1148/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 5de13a784fcec95613f80f509a20db29c6e90a66 into e80b3ddccb3df636be98522cba70692b968dab1c )

File: package/SUSEConnect.spec [hunk 1]
```diff
@@ -26,7 +26,6 @@ Requires:       zypper >= 1.11.32
 Conflicts:      suseRegister, yast2-registration < 3.1.129.7
 
 Obsoletes:      ruby2.1-rubygem-suse-connect < %{version}
-Provides:       %{rb_default_ruby_suffix}-rubygem-suse-connect = %{version}
```
Comment (lagartoflojo) [inline package/SUSEConnect.spec]: So we don't need this anymore?

Comment (sandrew) [inline package/SUSEConnect.spec]: We discussed it with @digitaltom and then I tested this rpm against old two-package configuration, and it turns out to be redundant

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-Connect-Prophet/5461/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 5de13a784fcec95613f80f509a20db29c6e90a66 into a02edebe7825428f8e0a5d20bf3875166f847c87 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-Connect-Prophet/10707/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 3fdcff4d7f083f35235ca8a3fa2422b9ed20489e into a02edebe7825428f8e0a5d20bf3875166f847c87 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-Connect-Prophet/13326/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 096e4b3ea3b806116fcde92efafe6d112cfce22c into a02edebe7825428f8e0a5d20bf3875166f847c87 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-Connect-Prophet/13506/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged def24574e4a2d5c245a753ba4bb246b1898caa2a into a02edebe7825428f8e0a5d20bf3875166f847c87 )

**Pull request merged**